src/tools/index.md

```diff
@@ -130,6 +130,10 @@ Here are the general-purpose tools you might want to install:
 [dart_style]: {{site.pub-pkg}}/dart_style
 [dartfix]: {{site.pub-pkg}}/dartfix
 
+### Debugging and diagnostics
+
+[Dart DevTools](/tools/dart-devtools)
+: A suite of suite of debugging and performance tools.
```

Comment:
suite of ~~suite of~~

Reply:
We should probably say that they're useful for both Flutter and command-line apps.

Reply:
Cal you elaborate? We already say "for Dart and Flutter", and have subsections below for Flutter and console...

Reply:
I see, you added those subsections in your edits. That looks good; resolving this comment.